api/Cargo.toml

```diff
@@ -4,9 +4,15 @@ version = "0.1.0"
 edition = "2021"
 
 [dependencies]
+anyhow = { version = "1.0.71", features = [ "backtrace" ] }
 bitcoin = { version = "0.29.2", features = [ "serde" ] }
 serde = { version = "1.0", features = [ "derive" ] }
 serde_json = "1.0"
+lightning = { version = "0.0.115", features = ["max_level_trace", "_test_utils"] }
```

Comment:
I want to try to keep the API crate simple. Perhaps we can even generate it from an OpenAPI document. So lets not add lightning dependency here.

Reply:
Relate to #401

Reply:
https://github.com/kuutamolabs/lightning-knd/issues/401#issuecomment-1594293555
I already let it go, else you will see PR not issues, but the `use api::lightning::*` is the side effect from #401.

Reply:
Is this conversation resolved? @JosephGoulden ?

Reply:
Could you open a PR to delete the `api` crate first? Such that this PR can base on that, and we do not need to wait for the swagger spec PR.

Reply:
The swagger spec PR will generate the code in the API crate. And its probably just going to be a string, there won't be any strong typing on deserialisation. We have to do that validation in our API server. Maybe NetAddress has to live in the settings crate for now?

Reply:
Okay, let me do it.